sql3/test/defs/defs_string_functions.go

```diff
@@ -820,5 +820,86 @@ var stringScalarFunctionsTests = TableTest{
 			),
 			ExpErr: "[0:0] value '-1' out of range",
 		},
+		{
+			name: "FormatString",
+			SQLs: sqls(
+				"select format('this or %s', 'that')",
+			),
+			ExpHdrs: hdrs(
+				hdr("", fldTypeString),
+			),
+			ExpRows: rows(
+				row(string("this or that")),
+			),
+			Compare: CompareExactOrdered,
+		},
+		{
+			name: "FormatBoolean",
+			SQLs: sqls(
+				"select format('is this %t?', true)",
+			),
+			ExpHdrs: hdrs(
+				hdr("", fldTypeString),
+			),
+			ExpRows: rows(
+				row(string("is this true?")),
+			),
```

Comment:
some of these tests are missing the line:
```
Compare: CompareExactOrdered,
```
without a `Compare` type specified for non-error tests, the tests aren't getting exercised.